static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner-parameters.js

```diff
@@ -18,17 +22,39 @@ const engagementBannerCopy = (): string =>
     everyone who reads our reporting, who likes it, helps fund it, our future would be much more secure.`;
 
 // Prices taken from https://membership.theguardian.com/<region>/supporter
+
+const paymentAmounts = {
```

Comment:
why is this in the banner params rather than geolocation? doesn't it apply to the epic too?

Reply:
OK, I see - if we're promoting membership. i.e., just in Australia, and not for long

Reply:
As discussed offline, there is more refactoring that we can do but we need to wait until AU is done. It will get simpler.